src/Context/FeatureContext.php

```diff
@@ -319,6 +324,56 @@ private static function get_behat_internal_variables() {
 		return $variables;
 	}
 
+	/**
+	 * Download and configure a copy of the SQLite plugin. This will add the db.php dropin to our cached
+	 * copy of WordPress files that are used for each scenario
+	 */
+	private static function download_and_configure_sqlite_plugin( $dir ) {
+		$download_url      = 'https://downloads.wordpress.org/plugin/sqlite-database-integration.zip';
+		$download_location = $dir . '/wp-content/plugins/sqlite-database-integration.zip';
+		Process::create(
+			Utils\esc_cmd(
+				'curl -sSfL %1$s > %2$s && chmod +x %2$s',
```

Comment:
Looks like this is copied from the phar method. I don't think we need the `chmod` part here.